### PR TITLE
JWT access strategy docs for Ory and Istio comparison

### PR DESCRIPTION
**Pull request by videlov**
**Description**

Changes proposed in this pull request:

- JWT access strategy documentation for Ory and Istio and comparison

**Related issue(s)**

https://github.com/kyma-project/api-gateway/issues/334

### CODE REVIEW

File: docs/jwt-ory-to-istio.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+  rules:
+    - path: /.*
+      methods: ["GET"]
+      mutators: []
+      accessStrategies:
+        - handler: jwt
+          config:
+            authentications:
+            - issuer: $ISSUER
+              jwksUri: $JWKS_URI
```
Comment (triffer) [inline docs/jwt-ory-to-istio.md:74]: In the Oathkeeper example we used `$JWKS_URI1`, maybe we should use the same name here.

File: docs/jwt-ory-to-istio.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+              fromHeaders:
```
Comment (triffer) [inline docs/jwt-ory-to-istio.md:75]: Why do we have the additional configuration like fromHeaders, fromParameters, requiredScopes or audiences only in the Istio and not in the Oathkeeper example? Should we have it this detailed here at all?

Comment (videlov) [reply docs/jwt-ory-to-istio.md:75]: Yes, this makes sense will adapt it. Thx.

File: docs/jwt-ory-to-istio.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+    port: 8080
+  rules:
+    - path: /.*
+      methods: ["GET"]
+      mutators: []
+      accessStrategies:
+        - handler: jwt
+          config:
+            authentications:
+            - issuer: $ISSUER
```
Comment (triffer) [inline docs/jwt-ory-to-istio.md:73]: In the Oathkeeper example we used `$ISSUER1`, maybe we should use the same name here.
```suggestion
            - issuer: $ISSUER1
```

File: docs/jwt-ory-to-istio.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+              jwksUri: $JWKS_URI
```
Comment (triffer) [inline docs/jwt-ory-to-istio.md:74]: In the Oathkeeper example we used $JWKS_URI1, maybe we should use the same name here.
```suggestion
              jwksUri: $JWKS_URI1
```

**Review by nataliasitko — APPROVED** [review]

**Pull request merged**